### DISCUSSION TITLE
欢迎来到Transformer讨论组！👋

### DISCUSSION THREAD

**Discussion by luo-zhan** [Robot-L/translator#3] [Q&A] [answered]
## 👋 欢迎！
  讨论组是用来你和其他小伙伴进行交流的地方，你可以：
  * 提出问题
  * 分享你的idea
  * 持开放态度，包容友好的进行交流

**Comment by JavaRookie** [comment]
现在实体类属性翻译还是轮询查表吗

**Reply by JavaRookie** [reply]
中央仓库没找到2.0.0-RELEASE版本

**Reply by luo-zhan** [reply]
> 中央仓库没找到2.0.0-RELEASE版本

有的啊，可能你用的镜像仓库没同步？

**Reply by luo-zhan** [reply]
> 现在实体类属性翻译还是轮询查表吗

暂时是这样，后续考虑增加缓存和异步查询功能

**Reply by luo-zhan** [reply]
v2.1.0已支持线程级别的转换缓存，相同数据转换直接从缓存取，避免重复io

**Reply by luo-zhan** [reply, marked as answer]
v2.3.1 已支持批量翻译，可以用@TransformFK方便进行外键字段翻译
